share/picongpu/tests/openPMD-viewer-test/lib/python/test/param-test.py

```diff
@@ -0,0 +1,235 @@
+import argparse
+import sys
+import numpy as np
+import scipy.constants as spc
+from openpmd_viewer import OpenPMDTimeSeries
+import openpmd_viewer
+
+print(openpmd_viewer.__version__)
```

Comment:
```suggestion
print("openPMD-viewer version:", openpmd_viewer.__version__)
```
Just to let the user see what the printed number actually means.